tests/cxx_std_macro/BUILD

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+load("//cc:cc_binary.bzl", "cc_binary")
+load("//cc/cxx_standard:cxx_standard.bzl", "cxxopts")
+
+licenses(["notice"])
+
+cc_binary(
+    name = "main",
+    srcs = ["main.cc"],
+    copts = cxxopts(default = "17"),
```

Comment:
This won't apply transitively. Could that become a usability issue when dealing with ABI-breaking standards changes (11 vs. 14)

Reply:
This wasn't intended to apply transitively. Yes it does mean you might specify this a lot, but in all the projects I've worked on when we get to a critical mass of cc targets folks have written macros to wrap this kind of boilerplate. But the intent was for this to be applied granularly.